node/lib/task.ts

```diff
@@ -119,27 +119,55 @@ var locStringCache: {
 } = {};
 var resourceFile: string;
 var libResourceFileLoaded: boolean = false;
+var resourceCulture: string = 'en-US';
+
+function loadResJson(resjsonFile: string): any {
+    var resJson: {} = null;
+    if (exist(resjsonFile)) {
+        var resjsonContent = fs.readFileSync(resjsonFile, 'utf8').toString();
+        // remote BOM
+        if (resjsonContent.indexOf('\uFEFF') == 0) {
+            resjsonContent = resjsonContent.slice(1);
+        }
+
+        try {
+            resJson = JSON.parse(resjsonContent);
+        }
+        catch (err) {
+            debug('unable to parse resjson with err: ' + err.message);
+            resJson = null;
+        }
+    }
+    else {
+        debug('.resjson file not found: ' + resjsonFile);
```

Comment:
It's your call, but one suggestion here: Most folks are not going to have a resjson file. I predict this "file not found" is going to confuse customers. Consider debug logging the file path if it is found, otherwise not debug log anything.